x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.test.tsx

```diff
@@ -135,7 +135,7 @@ describe('Alert details', () => {
     expect(alertDetails.queryByTestId('alertDetails')).toBeTruthy();
     expect(alertDetails.queryByTestId('alertDetailsError')).toBeFalsy();
     expect(alertDetails.queryByTestId('alertDetailsPageTitle')).toBeTruthy();
-    expect(alertDetails.queryByTestId('alert-summary-container')).toBeTruthy();
+    expect(alertDetails.queryByTestId('alertDetailsTabbedContent')).toBeTruthy();
```

Comment:
Does it mean we will not see `alert-summary-container` anymore or can we check both `alert-summary-container` and `alertDetailsTabbedContent` here?

Reply:
updated tests in 1c25061873737610b24e7a141ec555517971ed52